esm/interface/node.js

```diff
@@ -66,7 +66,7 @@ export class Node extends EventTarget {
     const ownerDocument = this.nodeType === DOCUMENT_NODE ?
                             this : this.ownerDocument;
     if (ownerDocument) {
-      const base = ownerDocument.querySelector('base');
```

Comment:
`<base>` elements without a `href` attribute should be skipped here. The [algorithm](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#document-base-url) is a bit more complicated (allowing for relative `href`), but maybe that can be addressed more thoroughly in a future PR.